### ISSUE TITLE
More information about versions/commits for each homebrew

### ISSUE THREAD

**Issue by tiliarou** [noahc3/SDSetup#10]
Does all the homebrew app updates on the fly as soon as a commit is pushed ?
I mean, do you build all by yourself automatically on a build server ? Is there some app from release on github or you always make by yourself/your buildserver ?

Is there a way to show the version/commit being included in the zip (like when mouseover, display version of the homebrew app).

Thanks.

**Comment by noahc3** [comment]
Versions are shown [next to the name of the homebrew](https://puu.sh/BpzgX/b699f2dcec.png). If it's a release, the version from the release title is used, or if one is not supplied, the version from the tag is used, or the date the release was made. If it is a specific commit, the commit is specified with c.xxxxxxxx where xxxxxxxx is the first 8 chars of the commit hash.

Everything is updated and tested manually because:
* Commits pushed directly to repos may not be production ready and therefore should be tested (and full releases should be prioritized).
* File directories and structures of released archives might change so it isn't safe to assume they will be in the same format forever.

**Comment by tiliarou** [comment]
I missed that when checking the website the first time, sorry.
Makes more sense with details about each version/date/commit meaning. Yes, adding to FAQ is a good idea.

Just one thing on the releases: some of the hombrew are very outdated in realease but have many useful commits (bug fixed etc...)
Do you take the time to check this ? I'm thinking about the nx-hbmenu : 2.0 is from 28/02. Since then, there was 32 commits and the latest works fine (for me) on 5.1.
Same for NX-shell and appstore-nx (but I didn't try latest commits for the latter).

For Tinfoil, it says v1 but in fact commits are pushed several days a week, so wouldn't it be better to add these 2 informations ? (v1 | d3c12b1)

**Comment by noahc3** [comment]
In general if a dev does releases at all, I would prefer to wait for those as they will likely be more stable. If they don't do releases (ex. sys-ftpd) then builds are done directly from the repo. It's hard to test everything thoroughly so sticking to releases would offer the best user experience, usually. tomGER wants to wait on newer AppstoreNX since something is in progress. I'll check out NXShell.

The HBMenu version is actually just a placeholder. I needed to specify a version for caching purposes but I couldn't really track it back to a specific commit since it also contains patches from SciresM for general compatibility which wasn't included in the repo at the time. Tinfoil is specifically commit d3c12b1d but I forgot to tell tomGER to specify that as the version when he updated it.

CFWs have versions listed on the Credits page. I've been meaning to add them to the buttons though, I'll get around to that eventually. Atmosphere has a date as the version as it is a build from the latest commit available then but also includes extras that differs it enough from the repo that it wouldn't really be the same.

tl;dr I'll update hbmenu and fix the Tinfoil version, as well as add versions to the CFW buttons.

**Comment by tiliarou** [comment]
Thanks a lot ! Will close the PR as all information is already pretty much there or will be added eventually.

**Comment by noahc3** [comment]
These issues are now fixed by commits 02f4ef197d1597ac22d88ed7d605268d9b461cfa 063e43727b92391ae0727b2258cdffa474f86614 and 251c6ae853470b1157771983d3dd8aeab52d8345. These changes will be live soon.
